src/modules/cp/components/ApproveRequestBlade/ApproveRequestBlade.const.ts

```diff
@@ -0,0 +1,25 @@
+import { format } from 'date-fns';
+import { i18n } from 'next-i18next';
+import { date, object, SchemaOf, string } from 'yup';
+
+import { getLocaleFromi18nLanguage } from '@shared/utils';
+
+import { ApproveRequestFormState } from './ApproveRequestBlade.types';
+
+export const APPROVE_REQUEST_FORM_SCHEMA = (): SchemaOf<ApproveRequestFormState> => {
+	return object({
+		accessFrom: date().required(i18n?.t('Een startdatum is verplicht.')),
+		accessTo: date().required(i18n?.t('Een einddatum is verplicht.')),
+		accessRemark: string().optional(),
+	});
+};
+
+export const ApproveRequestAccessDateFormatter = (date?: Date): string => {
```

Comment:
const gaan we meestal in upper snake case definiëren APPROVE_REQUEST_ACCESS_DATE_FORMATTER maar dit lijkt me eerder een helper func, dat mag dan in `ApproveRequestBlade.utils.ts` als `formatApproveRequestAccessDate`. Als deze op meerdere plaatsen nodig zijn mag dat in de `shared/utils`